docs/toc.json

```diff
@@ -86,7 +86,9 @@
 			{ "docs/designers-developers/developers/tutorials/block-tutorial/introducing-attributes-and-editable-fields.md": [] },
 			{ "docs/designers-developers/developers/tutorials/block-tutorial/block-controls-toolbar-and-sidebar.md": [] },
 			{ "docs/designers-developers/developers/tutorials/block-tutorial/creating-dynamic-blocks.md": [] },
-			{ "docs/designers-developers/developers/tutorials/block-tutorial/generate-blocks-with-wp-cli.md": [] }
+			{ "docs/designers-developers/developers/tutorials/block-tutorial/generate-blocks-with-wp-cli.md": [] },
```

Comment:
How about shifting the order and having the inner blocks tutorial first? The steps of the block tutorial look clearer that way (build more complex things as you advanced) and the rhythm doesn't get broken by the CLI tutorial (which isn't about things you could use to build blocks but block set up instead).

Reply:
I'm wondering if this CLI tutorial is still up to date 🤔

Reply:
Yeh, I kinda want to remove the CLI suggestion. I was going to post about that in a separate PR. I'm not sure if the scaffolding that it generates is really what we want.

Reply:
It may be worth revisiting the CLI part, even to delete it. In the meantime, I think that section fits better after the nested/inner blocks tutorial.